draft-ietf-rats-corim.md

```diff
@@ -1254,11 +1254,28 @@ applies to all measurements in the triple, including those in `measurement-value
 # CoBOM {#sec-cobom}
 
 A Concise Bill of Material (CoBOM) object represents the signal for the
-Verifier to activate the listed tags. Data contained in a tag MUST NOT be used
+Verifier to activate the listed tags. Data contained in a tag SHOULD NOT be used
```

Comment:
The conditions under which the recommendation can be skipped should be listed adjacent to the `SHOULD NOT`.

Reply:
I think, I have made it clear that when CoBOMs are required, then this should apply.

If we read the entire para, it makes more clarity on the context!